### PR TITLE
Add missing dependency

### PR DESCRIPTION
**Pull request by AMDmi3**
Fixes link error:

    ...omim/search/intermediate_result.cpp:44: undefined reference to OSMTimeRange::OSMTimeRange...

### CODE REVIEW

**Comment by mapsmetest** [comment]
Jenkins asks: can anyone have a look at this request?
If you want Jenkins to compile and test this, write "test this please" or JT_TP (without the underscore in the middle).

**Comment by ygorshenin** [comment]
We do not compile opening_hours for linux, please, add this dependency conditionally, for freebsd only.

**Comment by AMDmi3** [comment]
Why don't you build it? Likely it should be the same for FreeBSD.

**Comment by ygorshenin** [comment]
The problem is that libopening_hours can't be built with linux clang3.5 due to missing codecvt header. And by default clang3.5 is shipped with Debian (Jessie) and CentOS. Of course, there exists a workaround - manually download and build clang3.6 with libc++, but this is a headache. Is it possible to add this dependency conditionally for freebsd only?

**Comment by AMDmi3** [comment]
Shouldn't it be ```!linux*``` instead?

**Comment by ygorshenin** [comment]
Yes, I think this is correct. @syershov WDYT?

**Comment by syershov** [comment]
I think same.

**Comment by ygorshenin** [comment]
OK, @AMDmi3, could you please add this check?

**Comment by AMDmi3** [comment]
Done

**Comment by ygorshenin** [comment]
LGTM

File: routing/routing_integration_tests/routing_integration_tests.pro [hunk 1]
```diff
@@ -13,6 +13,11 @@ TEMPLATE = app
 ROOT_DIR = ../..
 DEPENDENCIES = map routing search storage indexer platform geometry coding base osrm jansson protobuf tomcrypt succinct stats_client
 
+# this dependency is not built on Linux
```
Comment (mpimenov) [inline routing/routing_integration_tests/routing_integration_tests.pro]: Зачем этот комментарий?

Comment (AMDmi3) [inline routing/routing_integration_tests/routing_integration_tests.pro]: It is obvious, no? To explain why this dependency is not enabled on Linux. That is not obvious at all.

Comment (ygorshenin) [inline routing/routing_integration_tests/routing_integration_tests.pro]: But your comment doesn't explain why this dependency is not enabled on Linux. It just claims that opening_hours is not built on Linux.

Comment (AMDmi3) [inline routing/routing_integration_tests/routing_integration_tests.pro]: And it shouldn't. Why opening_hours is not build on Linux should be explained in 3party/3party.pro

**Pull request merged**